content/blog/esp32-bootstrapping/index.md

```diff
@@ -0,0 +1,375 @@
+I (226) heap_runtime: ESP heap runtime init at 0x3fcab850 size 247 kB.
+
+
+*** Booting Zephyr OS build v3.7.0-5030-ge7db0f8aff81 ***
+uart:~$
+```
+
+
+### MCUboot Zephyr RTOS port (ZP)
+
+First lets take a look at how to manually build the MCUboot and the subsequent application. Each `west flash` in the code is using its own flash partition and it is not overwritten by each other. 
+
+Building MCUboot separately at its location:
+```shell
+cd zephyrproject/bootloader/mcuboot
+west build -b esp32s3_devkitm/esp32s3/procpu boot/zephyr -p
+west flash && west espressif monitor
+```
+
+Building the sample application that is loadable by the MCUboot created in previous step:
+```shell
+cd zephyrproject/zephyr
+west build -b esp32s3_devkitm/esp32s3/procpu samples/net/wifi -p -DCONFIG_BOOTLOADER_MCUBOOT=y
+west flash && west espressif monitor
+```
+
+Now, we can rely on the Sysbuild and build all images at once.
+
+Building the application with MCUboot(ZP) using sysbuild:
+```shell
+cd zephyrproject/zephyr
+west build -b esp32s3_devkitm/esp32s3/procpu samples/new/wifi -p --sysbuild
+west flash && west espressif monitor
+```
+
+In both cases (manual & sysbuild) we should see the following console output after the image boots:
+```
+ESP-ROM:esp32s3-20210327
+Build:Mar 27 2021
+rst:0x1 (POWERON),boot:0xa (SPI_FAST_FLASH_BOOT)
+SPIWP:0xee
+mode:DIO, clock div:2
+load:0x3fc946c8,len:0x36dc
+load:0x40374000,len:0x106a8
+entry 0x4037d24c
+I (98) boot: ESP Simple boot
+I (98) boot: compile time Aug 23 2024 09:58:26
+W (98) boot: Unicore bootloader
+I (98) spi_flash: detected chip: gd
+I (99) spi_flash: flash io: dio
+W (102) spi_flash: Detected size(8192k) larger than the size in the binary image header(2048k). Using the size in the binary image header.
+I (115) boot: chip revision: v0.2
+I (117) boot.esp32s3: Boot SPI Speed : 40MHz
+I (121) boot.esp32s3: SPI Mode       : DIO
+I (125) boot.esp32s3: SPI Flash Size : 8MB
+I (129) boot: Enabling RNG early entropy source...
+I (134) boot: DRAM: lma 0x00000020 vma 0x3fc946c8 len 0x36dc   (14044)
+I (140) boot: IRAM: lma 0x00003704 vma 0x40374000 len 0x106a8  (67240)
+I (146) boot: padd: lma 0x00013db8 vma 0x00000000 len 0xc240   (49728)
+I (152) boot: IMAP: lma 0x00020000 vma 0x42000000 len 0x52b68  (338792)
+I (159) boot: padd: lma 0x00072b70 vma 0x00000000 len 0xd488   (54408)
+I (165) boot: DMAP: lma 0x00080000 vma 0x3c060000 len 0x1481c  (83996)
+I (171) boot: Image with 6 segments
+I (174) boot: DROM segment: paddr=00080000h, vaddr=3c060000h, size=14820h ( 84000) map
+I (182) boot: IROM segment: paddr=00020000h, vaddr=42000000h, size=52B66h (338790) map
+I (226) heap_runtime: ESP heap runtime init at 0x3fcab850 size 247 kB.
+
+
+*** Booting Zephyr OS build v3.7.0-5030-ge7db0f8aff81 ***
+uart:~$
+```
+
+### MCUboot Espressif port (EP)
+
+How-to build MCUboot Espressif port [article](https://docs.mcuboot.com/readme-espressif.html)
```

Comment:
This part looks unfinished. If you want just to mention the article, I suggest.

`To learn how to build the MCUboot Espressif Port, check out this [article](https://docs.mcuboot.com/readme-espressif.html).`